controller/AuthControll.js

```diff
@@ -0,0 +1,68 @@
+const User = require('../models/user')
+const bcrypt = require('Bcryptjs')
+const jwt = require('jsonwebtoken')
+require('dotenv').config()
```

Comment:
This required only in your initial file, once.
No need to require it again elsewhere